e2e/Makefile

```diff
@@ -151,4 +159,69 @@ setup:
 	curl -o $(BINDIR)/kubectl -sfL https://storage.googleapis.com/kubernetes-release/release/v$(KUBERNETES_VERSION)/bin/linux/amd64/kubectl
 	chmod a+x $(BINDIR)/kubectl $(BINDIR)/kind
 
-.PHONY: launch-kind shutdown-kind start-lvmd stop-lvmd test clean setup
```

Comment:
Could you rewrite `PHONY` for each target like below.

```
.PHONY: clean
clean:
    ~~~

.PHONY: setup
setup:
    ~~~~
```

Reply:
fixed it.